deps/verifier/src/snp/mod.rs

```diff
@@ -307,6 +310,38 @@ fn get_common_name(cert: &x509::X509) -> Result<String> {
     Ok(e.data().as_utf8()?.to_string())
 }
 
+// Function to request vcek from KDS. Return vcek in der format.
+fn request_endorsement_key_kds(
```

Comment:
nit: `request_endorsement_key_kds` -> `get_endorsement__key_from_kds`

Reply:
Per Claudio's suggestion, I've renamed it to fetch_vcek_from_kds().  If appropriate, please mark this comment as Resolved.